extensions/placeholder/package.json

```diff
@@ -107,41 +107,85 @@
               "value": "100"
             }
           ]
-        },
-        {
-          "name": "applyTo",
-          "title": "Apply Wallpaper To",
-          "description": "Apply the wallpaper to which monitor. Valid only when you are using an extended monitor.",
-          "type": "dropdown",
-          "data": [
-            {
-              "title": "Current Monitor",
-              "value": "current"
-            },
-            {
-              "title": "Every Monitor",
-              "value": "every"
-            }
-          ],
-          "required": false,
-          "default": "every"
         }
       ]
     },
     {
-      "name": "random-placeholder-image",
-      "title": "Random Placeholder Image",
+      "name": "random-placeholder",
```

Comment:
Can we keep the old name here? It's the unique ID of the command where ranking, aliases and hotkeys are saved